pkg/services/sqlstore/dashboard_test.go

```diff
@@ -442,7 +449,7 @@ func TestDashboard_SortingOptions(t *testing.T) {
 	})
 }
 
-func insertTestDashboard(title string, orgId int64, folderId int64, isFolder bool, tags ...interface{}) *models.Dashboard {
+func insertTestDashboard(title string, orgId int64, folderId int64, isFolder bool, tags ...interface{}) (*models.Dashboard, error) {
```

Comment:
I think with test utility functions, it's better to make them take a `*testing.T` first argument and use `require.NoError(t, err)` etc. Also stick a `t.Helper()` call at the beginning of the function. You're in the context of a test, so you might as well use the `testing.T` facilities.